setup_bibliography.Rmd

```diff
@@ -0,0 +1,58 @@
+---
+title: "Bibliography Setup"
+author:
+  name: "Thomas Roppelt"
+  affiliation: "Friedrich-Alexander-Universität Erlangen-Nürnberg (FAU)"
+date: "`r format(Sys.time(), '%d %B, %Y')`"
+bibliography: library.bib
+---
+
+```{r setup, echo = FALSE}
+knitr::opts_chunk$set(echo = FALSE)
+```
+
+## Bibliography Managers
+Bibliography managers are important to keep all the literature you work with well organized.
+This page will help you understand pros and cons of different bibliography managers as well as help you set up Zotero, a free and open-source bibliography manager.
+
+## Comparison
+There are many different bibliography managers available, e.g. Citavi, Colwiz, EndNote, JabRef, Mendeley, Zotero and ZoteroBib.
+Since we focus on open-source software, software like Citavi and EndNote are out of question.
+Of the named software, only JabRef, Zotero and ZoteroBib are free and open source.
+For a detailed feature comparison, you may want to look [here (german)](https://ub.fau.de/schreiben-publizieren/literaturverwaltung/) or [here (english)](https://en.wikipedia.org/wiki/Comparison_of_reference_management_software).
+While the following paragraphs will cover Zotero, please feel free to use any of the other open source bibliography managers. As long as they can handle the [BibTex format](https://de.wikipedia.org/wiki/BibTeX) it should be all fine.
+
+## Zotero setup
+To get started with Zotero, there are some components that should be set up:
+
+### Zotero itself
+Windows: 
```

Comment:
maybe use the list markup for the different operating systems, or a description list as per pandoc's markdown.